Table creation for daopunks NFT traits

ebertbm opened the pull request:
Add DaoPunks traits table to be able to query the metadata of that NFT project

I've checked that:

* [x] the query produces the intended results
* [x] the folder name matches the schema name
* [x] the schema name exists in Dune
* [x] views are prefixed with `view_`, functions with `fn_`.
* [x] the filename matches the defined view, table or function and ends with .sql
* [x] each file has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`

masquot commented:
Thank you for adding daopunks NFT traits @ebertbm. The file you submitted seems to be cut off at line 1126. Can that be? You're missing the concluding `COMMIT` statement and maybe some lines with traits.

Also, I recommend making `token_id` a `PRIMARY KEY` instead of `NOT NULL` in the `INSERT TABLE` statement.

ebertbm commented:
Thanks @masquot for the feedback. I just added the primary key, also added an index on token_id as I saw there was an index on the traits files in the repo.  And also added the commit at the end of the file.

In total, there are 1111 tokens, so that's the correct number of lines.

masquot commented:
Great, thank you @ebertbm. DaoPunks traits are now available in Dune :rocket:
